src/main/fc/fc_rc.c

```diff
@@ -268,18 +279,18 @@ FAST_CODE uint8_t processRcSmoothingFilter(void)
     static FAST_RAM_ZERO_INIT uint16_t defaultDerivativeCutoffFrequency;
     static FAST_RAM_ZERO_INIT uint16_t filterCutoffFrequency;
     static FAST_RAM_ZERO_INIT uint16_t derivativeCutoffFrequency;
-    static FAST_RAM_ZERO_INIT uint8_t interpolationChannels;
 
     if (!initialized) {
         initialized = true;
-        interpolationChannels = rxConfig()->rcInterpolationChannels + 2; //"RP", "RPY", "RPYT"
         filterCutoffFrequency = rxConfig()->rc_smoothing_input_cutoff;
         derivativeCutoffFrequency = rxConfig()->rc_smoothing_derivative_cutoff;
     }
 
     if (isRXDataNew) {
-        for (int i = 0; i < interpolationChannels; i++) {
-            lastRxData[i] = rcCommand[i];
+        for (int i = 0; i <= THROTTLE; i++) {
```

Comment:
Maybe introduce a macro/enum called `PRIMARY_CHANNEL_COUNT`, set it to `throttle + 1` and use instead of `<=`?